stream/personalization.py

```diff
@@ -0,0 +1,122 @@
+class Personalization(object):
+    def __init__(self, client, token):
+        """
+
+        :param client: the api client
+        :param token: the token
+        """
+
+        self.client = client
+        self.token = token
+
+    def get(self, url, **params):
+        """
+        Get personalized activities for this feed
+        :param url: personalized url endpoint i.e "follow recommendations"
```

Comment:
I think `url` is very confusing (bit too internal for a customer facing method). This is a REST client and we are going to use this parameter to select a different resource right?

Reply:
how about `resource`?